### PR TITLE
Add kill/death tracker to acex

### PR DESCRIPTION
**Pull request by PabstMirror**
Redo https://github.com/acemod/ACE3/pull/4229 as an addon for acex

Will show kills at mission end screen.
Attempts to ID killer when unit's death was scripted.

Example below, note the vanilla kill system only shows 2 kills where as this system shows 5.
x

The last damage source isn't going to be perfect; e.g. get shot once, and then fall out of a chopper 30 minutes later.

### CODE REVIEW

File: addons/killtracker/XEH_postInit.sqf [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+        private _killerSide = if (!isNull (group _killer)) then {side group _killer} else {side _killer};
+        if ([_unitSide, _killerSide] call BIS_fnc_areFriendly) then {
+            _killInfo pushBack "<t color='#ff0000'>Friendly Fire</t>";
+        };
+    };
+
+    // Log bleed out - ToDo: could change setDead to log the specific medical cause (e.g. blood loss / cardiac arrest / overdose)
+    private _bloodVolume = _unit getVariable ["ace_medical_bloodVolume", 100];
+    if (_bloodVolume <= 60) then {
+        _killInfo pushBack format ["Blood %1%2", floor _bloodVolume, "%"];
```
Comment (Glowbal) [inline addons/killtracker/XEH_postInit.sqf]: The `"%"` could be included in the first string.

**Comment by SilentSpike** [comment]
Question: What is the `0_clean` in the top left of the screenshot?

**Comment by esteldunedain** [comment]
> Question: What is the 0_clean in the top left of the screenshot?

Probably the mission name

**Comment by SilentSpike** [comment]
Ah, good call :+1:

File: addons/killtracker/XEH_postInit.sqf (hunk 1 above)
Comment (PabstMirror) [inline addons/killtracker/XEH_postInit.sqf]: I think I have done it before, but I forgot how to include % inside `format`s.
Tried `\%` and `%%`

**Comment by ElTyranos** [comment]
Is there a way to send all this to RPT file ?

**Comment by PabstMirror** [comment]
I guess I could make an ace_setting that would log this stuff to rpt on mission end

**Comment by ElTyranos** [comment]
Would be awesome ! My scripting ability is very limited 😭

**Pull request merged**